tests/system/base.py

```diff
@@ -370,19 +370,20 @@ def _verify_run_outputs(
         loss: typing.Optional[int] = None,
         best_iteration: typing.Optional[int] = None,
         iteration_results: bool = False,
+        fragment: str = "",
```

Comment:
```suggestion
        iteration: typing.Optional[int] = None,
```
and If iteration is not None, it will prepend the value with a #